doc/source/user_guide/pymc3_refitting_xr_lik.ipynb

```diff
@@ -6,14 +6,14 @@
    "source": [
```

Comment:
<p>The are many instances of "{class}" and "{func}" in the text. Other than that It LGTM</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz/pull/1800/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
these are rendered as links in the output: https://arviz--1800.org.readthedocs.build/en/1800/user_guide/pymc3_refitting_xr_lik.html. Should there be less links? i.e. maybe only the first occurrence of the function/class?

Reply:
ohh I see, but If someone wants to download and run the notebooks they will see the raw syntax and not the links. There is a way to have both?